sufia-models/app/models/concerns/sufia/works/generic_work.rb

```diff
@@ -5,7 +5,9 @@ module GenericWork
 
     include Sufia::Works::Work
     include Sufia::Works::CurationConcern::WithBasicMetadata
-    include Sufia::GenericFile::Permissions
+    include Sufia::GenericFile::Batches
```

Comment:
Otherwise, for a work shown in collection work list, it will complain two methods (content and processing?) not found.